# this packs the sequence the robot identity an the time stamp
Header header

# Navigation data of aria robots
float64  TransRatio
float64  RotRatio
float64  LatRatio
float64  MaxVel
